package.json

```diff
@@ -64,12 +64,12 @@
     "express-ipfilter": "0.3.1",
     "express-rate-limit": "3.2.0",
     "google-libphonenumber": "^3.2.25",
-    "graphql": "14.7.0",
+    "graphql": "15.8.0",
```

Comment:
the troubles started here when I wanted to bump into `grapqhl` 15. The first `graphql-tools` version that allowed that, was 4.0.8. 
Unfortunately, that version brought many issues along with it from broken enums logic, different file structure, removed helpers. You can see me working on making adjustments all over the place now to satisfy the new API.
You can see me working on that here in this commit and this one https://github.com/artsy/metaphysics/pull/5838/commits/bfc66a7c4c47e1acf237876635b8898c343aff91

I also had to make adjustments to the peer dependencies later in the funnel in order to remove the resolution I made. This is the commit for it https://github.com/artsy/metaphysics/pull/5838/commits/faf692583f1b7e34434fb2076d0d9bd759b01dc3

▶️ **Next** [Fix peer dependencies](https://github.com/artsy/metaphysics/pull/5838/commits/77b611d5e1d976313b8fd99fe52c357ddf553837)